update-reviewers.py

```diff
@@ -0,0 +1,200 @@
+    "reviewer_1": ["packages-reviewed", "reviewer"],
+    "reviewer_2": ["packages-reviewed", "reviewer"],
+    "editor": ["packages-editor", "editor"],
+    "submitting_author": [
+        "packages-submitted",
+        ["maintainer", "submitting-author"],
+    ],
+}
+
+
+missing_contribs = []
+for pkgName, issue_meta in packages.items():
+    print(pkgName)
+    # Get review meta -
+    # start with reviewers and editor
+    for issue_review_role in ppl_types:
+        gh_user = (
+            packages[pkgName][issue_review_role]["github_username"]
+            .lower()
+            .strip()
+        )
+        # If they aren't already in contribs, add them
+        if gh_user not in contribs.keys():
+            print("Missing user", gh_user, "adding them now.")
+            contribs.update(process_contribs.add_new_user(gh_user))
+        # Get list of existing packages in their list
+        existing_contribs = contribs[gh_user]["contributor_type"]
+        # This might not get used??
+        contrib_type_yml = ppl_types[issue_review_role][0]
+        review_role = ppl_types[issue_review_role][1]
+
+        # If name is missing in issue, populate contributors file
+        if issue_meta[issue_review_role]["name"] == "":
+            packages[pkgName][issue_review_role]["name"] = contribs[gh_user][
+                "name"
+            ]
+
+        # TODO - current issue is packages-reviewed and editor are being duplicated
+        # some of it might be case sensitive but...
+        # TODO - david's contrib types are not correct either he's missing submitting author and maintainer
+        # Update the contrib file with review package name
+        try:
+            if pkgName.lower() not in [ac.lower() for ac in existing_contribs]:
+                contribs[gh_user][contrib_type_yml].append(pkgName)
+        except (AttributeError, TypeError) as AE:
+            contribs[gh_user][contrib_type_yml] = [pkgName]
+        # Update their contributor type role - this populates the website
+        if not contribs[gh_user]["contributor_type"]:
+            contribs[gh_user]["contributor_type"] = [review_role]
+        elif review_role not in contribs[gh_user]["contributor_type"]:
+            # Adding vs append() because the review_role can be a list
+            try:
+                contribs[gh_user]["contributor_type"] + review_role
+            except TypeError as te:
+                contribs[gh_user]["contributor_type"].append(review_role)
+
+        # Finally check that peer-review is in the list of types
+        if "peer-review" not in contribs[gh_user]["contributor_type"]:
+            contribs[gh_user]["contributor_type"].append("peer-review")
+
+    # Now parse through all_current_maintainers
+    # Note that the first maintainer may be the submitting author
+    # TODO - is there a way to combine this with the above workflow
+    # the difference is there is a list of maintainers
+    print("Now on to maintainers")
```

Comment:
THe only reason that this is in a different section (the functionality is the same) is that i may have several maintainers in a list. so the input format is different than the format for a reviewer_1 entry that has a single reviewer and github user name. but it seems to me i could combine all of this code potentially.